assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Validate New-User.Can create New-User with explicit properties in local machine
```powershell
Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 40
40:             $result = New-User -AccountStore Machine `

```
Validate New-User.Error on user exists
```powershell
Expected an exception, with FullyQualifiedErrorId 'ParameterArgumentValidationError,Microsoft.PowerShell.Commands.NewUserCommand' to be thrown, but the FullyQualifiedErrorId was 'InvalidValue,Microsoft.PowerShell.Commands.NewUserCommand'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1:112 char:22
    + ...    $exc = { New-User -Name TestUserNew -PasswordNotRequired -AccountS ...
    +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.AccountManagement\AccountManagement.User.Tests.ps1: line 112
112:             $exc = { New-User -Name TestUserNew -PasswordNotRequired -AccountStore Machine -ErrorAction Stop } | Should -PassThru -Throw -ErrorId "ParameterArgumentValidationError,Microsoft.PowerShell.Commands.NewUserCommand"
```